test/parallel/test-performance-timeline.js

```diff
@@ -0,0 +1,38 @@
+// This file may needs to be updated to wpt:
```

Comment:
You can use a `.mjs` file for this test as in https://github.com/nodejs/node/blob/master/test/parallel/test-child-process-fork-url.mjs or https://github.com/nodejs/node/blob/master/test/parallel/test-fetch.mjs to take benefit of the top level await.